lighthouse-core/lib/statistics.js

```diff
@@ -59,6 +59,43 @@ function getLogNormalDistribution(median, falloff) {
   };
 }
 
+/**
```

Comment:
the existing `getLogNormalDistribution` returns a function that can be called repeatedly with different values, but I think no code has ever used it like that. This new function just gives a percentile back directly, with named parameters for the control points so it's not just a mass of numbers.